packages/admin/cms-admin/src/dam/Table/license/useLicenseValidityInformation.tsx

```diff
@@ -0,0 +1,61 @@
+import { add, compareAsc, getUnixTime } from "date-fns";
+
+const currentDate = new Date();
+const thirtyDaysInSeconds = 60 * 60 * 24 * 30;
+
+interface UseLicenseValidityInformationProps {
+    durationFrom?: Date;
+    durationTo?: Date;
+}
+
+interface LicenseValidityInformation {
+    isNotValidYet: boolean;
+    expiresSoon: boolean;
+    isExpired: boolean;
+    isValid: boolean;
+    startDate?: Date;
+    expirationDate?: Date;
+}
+
+export const useLicenseValidityInformation = ({ durationFrom, durationTo }: UseLicenseValidityInformationProps): LicenseValidityInformation => {
```

Comment:
any reason why you name ist like a hook?

Reply:
Because I originally intended it to be a hook. But it actually isn't a hook so I changed it now